packages/react-internal/src/components/Stack/__snapshots__/Stack.test.tsx.snap

```diff
@@ -392,7 +392,7 @@ exports[`Stack renders horizontal Stack with a gap in rtl context correctly 1`]
           text-overflow: ellipsis;
         }
         & > *:not(:first-child) {
-          margin-right: 10px;
+          margin-left: 10px;
```

Comment:
how did this change happen?

Reply:
It seems like some of the packages hadn't been built. It should be good now.